digiwf-gateway/src/main/java/de/muenchen/oss/digiwf/gateway/configuration/SecurityConfiguration.java

```diff
@@ -44,8 +44,10 @@ public SecurityWebFilterChain clientAccessFilterChain(ServerHttpSecurity http) {
     http
         .securityMatcher(ServerWebExchangeMatchers.pathMatchers("/clients/**"))
         .authorizeExchange(authorizeExchangeSpec -> {
-          authorizeExchangeSpec.anyExchange().authenticated();
+            authorizeExchangeSpec.pathMatchers(HttpMethod.OPTIONS, "/clients/**").permitAll()
```

Comment:
Das erscheint mir gerade nicht richtig, da dies doch die Methode auf `OPTIONS` einschränkt und andere Methoden demnach raus sind. Müsste das nicht zusätzlich sein?

Hab's nicht getestet. Aber vielleicht reden wir nochmal drüber.

Reply:
Das sagt ja nur aus, dass Options ohne Auth erlaubt ist und man sich für den Rest authentifizieren muss.
In der Security-Chain unten ist das genau so gelöst.

Reply:
Ah okay, sorry, hab das andere übersehen. Dann passt es